server/utils.py

```diff
@@ -253,3 +254,23 @@ def generate_number_table(num):
     Used in models.Assignment.mysql_course_submissions_query
     """
     return ' UNION '.join('SELECT {} as pos'.format(i) for i in range(1, num + 1))
+
+def upload_file(file, container=None, name=None, prefix=None, overwrite=False):
+    """ A utility function to get around having to specify
+    what file extensions are allowed.
+    """
+    extension = get_file_extension(file)
```

Comment:
So, essentially, we're accepting any file extension? That's fine with me, but why bother with `allowed_extensions` at all? Does flask-cloudy require it?